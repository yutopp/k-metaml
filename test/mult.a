fun mult (x, n) = if n = 0
                  then <1>
                  else <~x * ~(mult x (n - 1))>;

val cube = <fn y => ~(mult <y> 3)>;

fun exponent (n) = <fn y => ~(mult <y> n)>;