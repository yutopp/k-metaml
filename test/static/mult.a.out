Search results:

Solution 1:
V:Type -->
unit
N:Int -->
0
